responses/06-add-labels.md

```diff
@@ -18,7 +18,7 @@ These initial labels are awesome, but most maintainers add (or delete) labels to
 
 1. On the top right side of this pull request view, find the section titled: "Labels".
 1. Click the gear icon.
-1. Notice the text field at the top says **Filter or create labels**.
+1. Notice the text field at the top says **Filter labels**.
```

Comment:
That text field no longer says "filter or create". Just "filter labels"

Reply:
![image](https://user-images.githubusercontent.com/38021615/77965505-fab3c380-7295-11ea-8e3b-41157d9791d8.png)